jcache/src/main/java/com/github/benmanes/caffeine/jcache/CacheProxy.java

```diff
@@ -552,6 +553,7 @@ private V removeNoCopyOrAwait(K key) {
   }
 
   @Override
+  @CanIgnoreReturnValue
   public boolean remove(K key, V oldValue) {
```

Comment:
The annotation also applies to some other methods in this class; they're not flagged likely because there's no test calling them. I'll add a separate commit for this.

Reply:
Thanks. Mostly it is called by the jsr107 tck where our tests are to increase the code coverage due to possible gaps. The tck is a dependency that is unzipped during the build to run its tests, so a bit indirect and wouldn't be flagged.